components/common/labels/useLabelerDefinition.ts

```diff
@@ -0,0 +1,39 @@
+import clientManager from '@/lib/client'
+import { AppBskyLabelerDefs, ComAtprotoLabelDefs } from '@atproto/api'
+import { useQuery } from '@tanstack/react-query'
+import { ExtendedLabelerServiceDef } from './util'
+
+export const useLabelerServiceDef = (did: string) => {
+  const { data: labelerDef } = useQuery({
+    queryKey: ['labelerDef', { did }],
+    queryFn: async () => {
+      const { data } = await clientManager.api.app.bsky.labeler.getServices({
+        dids: [did],
+        detailed: true,
+      })
+      if (!data.views?.[0]) {
+        return null
+      }
+
+      const labelerDef = data.views[0] as ExtendedLabelerServiceDef
+      if (labelerDef?.policies.labelValueDefinitions) {
+        const definitionsById: Record<
+          string,
+          ComAtprotoLabelDefs.LabelValueDefinition
+        > = {}
+        labelerDef.policies.labelValueDefinitions.forEach((def) => {
+          definitionsById[def.identifier] = def
+        })
+        labelerDef.policies.definitionById = definitionsById
+      }
```

Comment:
Is there something in particular that we find in here that we aren't able to get from `session.config.labeler`?

Reply:
so, the idea of this PR is that eventually, we could plug in other labelers into ozone. Bryan touched on this I think somewhere that other labelers may want to see labels added by bluesky. 

there's a bit more work left to get this all to work and get the backend to send labels from other labelers but once that's done, this will make more sense I think.